discord/lib/handlers/handle-render-authenticated.js

```diff
@@ -15,9 +15,9 @@
  * @param {express.NextFunction} next
  */
 export const handleRenderAuthenticated = (htmlDirectory, htmlFilename, request, response, next) => {
-  const { oauth, state } = request.session;
+  const { oauth } = request.session;
 
-  if (!oauth || !oauth.access_token || !state) {
+  if (!oauth || !oauth.access_token) {
```

Comment:
we don't need state to render the authenticated view

now that we're correctly initializing the authenticated session, state is consumed once auth is complete and doesn't remain on the request session object